### ISSUE TITLE
[Bug]: FTB Sky Odyssey 1.3.0 - Techreborn block causes game crash

### ISSUE THREAD

**Issue by GamingWithJanko** [FTBTeam/FTB-Modpack-Issues#746]
### Modpack

FTB Sky Odyssey

### Modpack version

1.3.0

### Log Files

CRASHLOG
https://paste.feed-the-beast.com/view/60f6eb2e

LOG
https://paste.feed-the-beast.com/view/773bbda8

DEBUG LOG
(Too big)

### Describe the bug

Game crashes upon placing "techreborn:plate_bending_machine", and after loading the world with said block placed.

### Steps to reproduce

1. Load new world (Optional creative)
2. Get block **techreborn:plate_bending_machine**
3. Place it
4. Crash the game

### Expected behaviour

The game freezes and crashes

### Additional information

MCEdit can remove the block.

NBTExplorer can't remove the block, because it gets reinjected for some odd reason.

**Comment by Raidobw2** [comment]
Thanks for the report! We haven't gotten to fix it yet, although the bug was known for a bit in the FTB Discord. Looks like I forgot to have a ticket here for it, do we'll use yours as a reminder for us later, thank you.
Here's a solution I can offer, which should work, let me know:

> You can update the mod that should fix the machine's code: I think https://www.curseforge.com/minecraft/mc-mods/techreborn/files/2966851 should be the one. Take the Tech Reborn mod from your mods folder and replace it with the new one.
> I don't think you'll need to update Reborn Core, but in case you crash, then you do: same thing https://www.curseforge.com/minecraft/mc-mods/reborncore/files/3330308.

**Comment by GamingWithJanko** [comment]
I'll try it out asap.

I'll notify ya here.

And you're very welcome! ^^

**Comment by GamingWithJanko** [comment]
The steps you put work.

But only AFTER removal the OLD Techreborn AND Reborncore, along with the cfg files of both.
Both jar files have been replaced and work as intended.

Suggestion: Maybe push that as an official update?

Greetings from germany!

**Comment by Raidobw2** [comment]
Yup, that's the plan haha. I should have time this summer to make an update for that pack, possibly other ones. Don't hesitate to let me know about any other problems you get on our packs, I'll try and sort them out everytime.
Cheers and have fun playing!

**Comment by GamingWithJanko** [comment]
Will do!

Thanks for helping me as well. :D
Have a good one.

I'll wait for you to close this ticket, unless told otherwise. :D

**Comment by GamingWithJanko** [comment]
Also... There's no route for peaceful players to finish the pack without slaying specific hostile mobs.

Can that be arranged in some way?

**Comment by Raidobw2** [comment]
Definitely, I can see about that this summer for peaceful players. And you can leave this ticket open, once I'll get to modpack updates (there's a ton of things to do before I get there though, for legacy content, but eventually I will) I'll actually use that ticket to update mods and fix that peaceful progression :)

**Comment by GamingWithJanko** [comment]
Neat! :D

I can already tell ya the following items:

Netherstar
Locks the final EMC tools (Project EX: Mainly all the Final generator items)
Locks the Ultimate speed upgrade (Extra utilities 2: In this case for the Evil infused stuffs)

That's all I have at this moment. If you got softlock items for me to add, let me know. :D
I check in here regularily.

Cheers from germany!

**Comment by TheonlyTazz** [comment]
Fixed the TechReborn by updating it, Update will be out soon, for the Peaceful player option please open up a new issue if thats still something we need/should look at
